build.Tests.ps1

```diff
@@ -0,0 +1,60 @@
+$here = Split-Path -Parent $MyInvocation.MyCommand.Path
+$sut = (Split-Path -Leaf $MyInvocation.MyCommand.Path) -replace '\.Tests\.', '.'
+# . "$here\$sut"
+$script = "$here\$sut"
+
+& $script -ErrorAction Stop -PowerShellModuleName PowerShellGet -PowerShellModuleVersion "2.1.3"
+
+Describe $script {
+  It "creates a module" {
+    Test-Path "import/powershellget/metadata.json" | Should -BeTrue
+  }
+
+  It "has a REFERENCE.md" {
+    Test-Path "import/powershellget/REFERENCE.md" | Should -BeTrue
+  }
+
+  It "has a type generated" {
+    Test-Path "import/powershellget/lib/puppet/type/dsc_psmodule.rb" | Should -BeTrue
+  }
+
+  Context "inside the module" {
```

Comment:
For these, do we want to leverage the `Invoke-PDKCommand` private function (we can dotsource it into the current scope for use outside the module) to do better introspection/success checking?

Reply:
done.